GiniVision/Classes/CameraViewController.swift

```diff
@@ -121,6 +121,10 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
         return FilePickerManager()
     }()
     fileprivate var detectedQRCodeDocument: GiniQRCodeDocument?
+    fileprivate var currentQRCodePopup: QRCodeDetectedPopupView? {
+        return self.view.subviews.flatMap { $0 as? QRCodeDetectedPopupView }.first
```

Comment:
You know how much I hate going through the view hierarchy in order to find something. If you need a view, hold a reference to it as you create it.

Reply:
Also, why would you have several?

Reply:
Yes, you are right. Although it is nothing in terms of performance to go through the view hierarchy with the `flatMap`, I think It is more clear if it is stored in the class (uglier though).